app/manager/session_manager.js

```diff
@@ -0,0 +1,29 @@
+const jwt = require('jwt-simple'),
+  config = require('./../../config'),
+  User = require('./../models').user;
+
+const SECRET = config.common.session.secret;
+
+exports.decode = toDecode => {
+  return new Promise((resolve, reject) => {
+    let decodedToken;
+    try {
+      decodedToken = jwt.decode(toDecode, SECRET);
+    } catch (err) {
+      reject(err);
+    }
+    resolve(decodedToken);
```

Comment:
couldnt you just resolve inside the try and remove the let declaration at the top?